src/libs/ReportUtils.js

```diff
@@ -1950,6 +1950,11 @@ function shouldReportBeInOptionList(report, currentReportId, isInGSDMode, iouRep
         return false;
     }
 
+    // Hide thread reports that haven't been commented on
+    if (isThread(report) && !report.lastMessageText) {
```

Comment:
Hi, this PR caused bug - ["Deleted message" chat appears in chat list for pinned deleted threads](https://github.com/Expensify/App/issues/22950#top)

More details here - https://github.com/Expensify/App/issues/22950#issuecomment-1645793420